blocks/carousel/carousel.js

```diff
@@ -20,4 +20,8 @@ export default function decorate(block) {
     buttons.append(button);
   });
   block.parentElement.append(buttons);
+  const button0 = document.getElementById('carousel-button-0');
```

Comment:
this should be recursive regardless the number of items in the carousel. I suggest to test how the whole carousel works with 1,2,3 or 4 items if everything works as expected

Reply:
Fixed.